src/game/state/gStates/gsMain.ts

```diff
@@ -99,20 +100,9 @@ export class MainGameState extends GameState {
 
   receiveInputText(input: string) {
     if (!store.getters.instantMode) {
-      if (/^\/help$/.test(input)) {
-        store.dispatch("addMessage", {
-          entity: "Help",
-          message:
-            "Press 'enter' to enter typed command mode, and 'escape' to get back to command mode." +
-            "\nYou can right-click on the map to see what is on that tile.",
-        });
-      } else {
-        // Handle as text command
-        store.dispatch("addMessage", {
-          entity: "Main state",
-          message: "Response to " + input,
-        });
-      }
+      const result = Parser.parse(input);
```

Comment:
Probably worth making this lowercase before parsing (I can't remember if it already is at this point)

Reply:
Good spot, I was originally doing that at one point but it must've got lost.